test/core/V3/GenArt721CoreV3_Events.test.ts

```diff
@@ -67,29 +69,373 @@ describe("GenArt721CoreV3 Events", async function () {
       .addProject("name", this.accounts.artist2.address);
   });
 
-  describe("ProjectCompleted", function () {
-    it("emits ProjectCompleted when being minted out", async function () {
-      for (let i = 0; i < this.maxInvocations - 1; i++) {
+  // describe("ProjectCompleted", function () {
+  //   it("emits ProjectCompleted when being minted out", async function () {
+  //     for (let i = 0; i < this.maxInvocations - 1; i++) {
+  //       await this.genArt721Core
+  //         .connect(this.accounts.artist)
+  //         .mint(
+  //           this.accounts.artist.address,
+  //           this.projectZero,
+  //           this.accounts.artist.address
+  //         );
+  //     }
+  //     // emits event when being minted out
+  //     expect(
+  //       await this.genArt721Core
+  //         .connect(this.accounts.artist)
+  //         .mint(
+  //           this.accounts.artist.address,
+  //           this.projectZero,
+  //           this.accounts.artist.address
+  //         )
+  //     )
+  //       .to.emit(this.genArt721Core, "ProjectCompleted")
+  //       .withArgs(this.projectZero);
+  //   });
+  // });
```

Comment:
Was this intended to be commented out?

Reply:
ah, thank you! unintentionally commented out, when it actually needed to be watching for the new event type. updated!

Reply:
actually, this was an old test for the old event that was replaced with the new ProjectUpdated one. Pushed another update that removed this test instead of commenting it out.